client/src/App.js

```diff
@@ -7,6 +7,7 @@ import DashBoard from "./pages/DashBoard";
 import SignIn from "./pages/SignIn";
 import SignUp from "./pages/SignUp";
```

Comment:
For the user picture, I think we should just go with a user icon instead of the picture in the spec. then once we implement file uploading you should be able to click on that icon and update pic. so i would delete that pic and add user icon. @milkcanbass

Reply:
Noted